lib/txgh/handlers/github/ping_handler.rb

```diff
@@ -0,0 +1,18 @@
+module Txgh
+  module Handlers
+    module Github
+      # Handles github's ping event, which is a test event fired whenever a new
+      # webhook is set up.
+      class PingHandler
+        include ResponseHelpers
+
+        def initialize(options = {})
```

Comment:
Yeah. Even though `PingHandler` doesn't inherit from `Handler`, it should conform to the same interface.